src/Operations.jl

```diff
@@ -955,21 +969,25 @@ function up(ctx::Context, pkgs::Vector{PackageSpec})
         pkg.version isa UpgradeLevel || continue
         level = pkg.version
         info = manifest_info(ctx.env, pkg.uuid)
-        if haskey(info, "repo-url")
-            pkg.repo = Types.GitRepo(info["repo-url"], info["repo-rev"])
-            new = handle_repos_add!(ctx, [pkg]; upgrade_or_add = (level == UPLEVEL_MAJOR))
-            append!(new_git, new)
-        else
-            ver = VersionNumber(info["version"])
-            if level == UPLEVEL_FIXED
-                pkg.version = VersionNumber(info["version"])
+        if info != nothing
```

Comment:
Is checking this a bug-fix? I.e. was it possible previously to have `info == nothing` here?

Reply:
Yes, if you do `up` without having a Manifest. But this is fixed on master.